src/Disks/DiskSelector.cpp

```diff
@@ -77,6 +77,9 @@ DiskSelectorPtr DiskSelector::updateFromConfig(
 
             /// TODO: Ideally ClickHouse shall complain if disk has changed, but
             /// implementing that may appear as not trivial task.
```

Comment:
This TODO looks irrelevant already (or it should be rewritten in another formulation)

P.S. sorry to bother, but you have conflicts with upstream, so you will need to fix the first anyway and this comment can be adjusted together

Reply:
Already fixed